### DISCUSSION TITLE
Allow defined auth/users columns to be returned with user data

### DISCUSSION THREAD

**Discussion by geodox** [nhost/nhost#71] [Ideas]
I've added some additional columns in the users table and it would be nice if I can automatically return that data along with the other default data that's returned when a user logs in.

**Comment by elitan** [comment]
We won't do this for the current version of Nhost. What I suggest you do is make an additional GraphQL request to get the data you need.

For Nhost v2 (https://waitlist.nhost.io/) we won't allow adding columns to the `users` table.
